### PR TITLE
Test Cards

### LINKED ISSUE #154: Article: Test cards

### PR DESCRIPTION
**Pull request by fanatiicx**
Closes #154 

## If an article related PR

- [x] I have checked that my suggestion follows the Contribution guide

- [x]  I have added which issue this PR closes

- [x] I have 0 warnings from Danger.js

- [x] If I have added images, they are in /files/ folder

## If other PR

- [ ] I have added which issue this PR closes

### CODE REVIEW

File: practices/test_cards.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
+---
+slug: test-cards
+stages:
+  - research
+short_description: A test card helps you to test and validate your business ideas by defining how you will test them, what you will measure, and what success looks like.
+tags:
+  - value proposition
```
Comment (prokopsimek) [inline practices/test_cards.md:7]:
```suggestion
  - test cards
  - value proposition
```

File: practices/test_cards.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+- Test cards help you to gather more information through testing and experimenting before sinking in too much money and resources in a business idea that might not be profitable.
+- Using test cards is a practical process that can save you the frustration of making business decisions based on guesses or following what others are doing without looking at the specific needs of your own customers. 
+
+## Problems the Test Card Helps to Solve
+
+- [Bad Product-Market Fit](/problems/bad_product_market_fit)
+- [Sunk Cost](/problems/sunk_cost)
+- [Unhappy Clients](/problems/unhappy_clients)
+- [Unnecessary Features](/problems/unnecessary_features)
+- [Unsuccessful Product](/problems/unsuccessful_products)
```
Comment (prokopsimek) [inline practices/test_cards.md:53]:
```suggestion
- [Bad Product-Market Fit](/problems/bad-product-market-fit)
- [Sunk Cost](/problems/sunk-cost)
- [Unhappy Clients](/problems/unhappy-clients)
- [Unnecessary Features](/problems/unnecessary-features)
- [Unsuccessful Product](/problems/unsuccessful-products)
```

**Review by prokopsimek — CHANGES_REQUESTED** [review]
@fanatiicx Thanks for the article! 👏

I have reviewed the article, proposed 2 suggestions that fix internal URLs, and added a keyword for SEO.

I am also not sure about those headlines. I'd suggest pluralizing all headlines and describe all sections as "test cards" instead of just singular "test card". E.g., `What Is a Test Card?` change to `What Are Test Cards?`. I need @ridlees opinion for this.

Also, the section `How to Implement...` is quite short, and I'd love to read more about the phase before the filling test card. What do I need to prepare before, and in what phase of a product are test cards valuable? When should I start testing with test cards, and how looks the process after the test card is filled? Should I start testing my hypothesis with people, or are these test cards just for our internal purposes?

Thanks, @fanatiicx! I welcome any other suggestions or discussions.

Please, @ridlees, could you also do a review? I am fine with the length of the article and just missing some more "sauce" there.

**Review by zeman-j4 — APPROVED** [review]

**Comment by fanatiicx** [comment]
@prokopsimek Good note about the plurality! I will make some edits to improve the article over the next week or two :)

**Comment by prokopsimek** [comment]
@fanatiicx It will be awesome! Thanks and let me know if you need any help. :)

**Comment by prokopsimek** [comment]
@fanatiicx Hello! How're you doing? Can I help you finish the article? 🙂

**Comment by fanatiicx** [comment]
@prokopsimek I would greatly appreciate it! It'd be a great way to learn from your edits.

I was not feeling well last week and will be travelling next week, hence I've been a bit slow with editing this piece. Thanks for your patience :)

**Review by fanatiicx — COMMENTED** [review]
I'm back from vacation and made the suggested changes. I would be happy to hear your feedback. Thanks!

**Comment by prokopsimek** [comment]
@fanatiicx Great job! Thank you for completing your PR. Awesome! 👏 😎

**Review by prokopsimek — APPROVED** [review]

**Comment by prokopsimek** [comment]
@fanatiicx https://developerexperience.io/practices/test-cards Thank you!
I have to fix contributors loading.

**Comment by fanatiicx** [comment]
@prokopsimek Thanks! 

By the way, I noticed a typo in the last section. Wasn't sure where to edit so I opened a [pull request](https://github.com/DXHeroes/knowledge-base-content/pull/373) for it with the changes made.

**Comment by prokopsimek** [comment]
@fanatiicx  Fixed, thanks!

**Pull request merged**